### PR TITLE
Added Token from Commander 2018

### PR DESCRIPTION
**Pull request by rikimaru1234321**

### CODE REVIEW

File: Mage.Client/src/main/resources/card-pictures-tok.txt [hunk 1]
```diff
@@ -367,6 +367,25 @@
 |Generate|TOK:C17|Vampire|||EdgarMarkovToken|
 |Generate|TOK:C17|Zombie||
 |Generate|TOK:C18|Myr|||BrudicladTelchorMyrToken|
+|Generate|TOK:C18|Angel|||AngelToken|
+|Generate|TOK:C18|Cat|||CatToken|
+|Generate|TOK:C18|Cat Warrior||
```
Comment (JayDi85) [inline Mage.Client/src/main/resources/card-pictures-tok.txt]: Space must be removed

**Comment by JayDi85** [comment]
File in root folder must be deleted

**Comment by JayDi85** [comment]
If you can build and run app then you can add that code to support download images from scryfall (check images/settings manually, maybe need fixes):

For `ScryfallImageSupportTokens.java`:
```
put("C18/Angel", "https://api.scryfall.com/cards/tc18/3/en?format=image");
put("C18/Beast", "https://api.scryfall.com/cards/tc18/13/en?format=image");
put("C18/Beast", "https://api.scryfall.com/cards/tc18/14/en?format=image");
put("C18/Cat Warrior", "https://api.scryfall.com/cards/tc18/15/en?format=image");
put("C18/Cat", "https://api.scryfall.com/cards/tc18/5/en?format=image");
put("C18/Clue", "https://api.scryfall.com/cards/tc18/19/en?format=image");
put("C18/Construct/1", "https://api.scryfall.com/cards/tc18/20/en?format=image");
put("C18/Construct/2", "https://api.scryfall.com/cards/tc18/21/en?format=image");
put("C18/Dragon Egg", "https://api.scryfall.com/cards/tc18/10/en?format=image");
put("C18/Dragon", "https://api.scryfall.com/cards/tc18/11/en?format=image");
put("C18/Elemental", "https://api.scryfall.com/cards/tc18/16/en?format=image");
put("C18/Horror", "https://api.scryfall.com/cards/tc18/22/en?format=image");
put("C18/Manifest", "https://api.scryfall.com/cards/tc18/1/en?format=image");
put("C18/Mask", "https://api.scryfall.com/cards/tc18/4/en?format=image");
put("C18/Myr/1", "https://api.scryfall.com/cards/tc18/23/en?format=image");
put("C18/Myr/2", "https://api.scryfall.com/cards/tc18/7/en?format=image");
put("C18/Plant", "https://api.scryfall.com/cards/tc18/17/en?format=image");
put("C18/Servo", "https://api.scryfall.com/cards/tc18/24/en?format=image");
put("C18/Shapeshifter", "https://api.scryfall.com/cards/tc18/2/en?format=image");
put("C18/Soldier", "https://api.scryfall.com/cards/tc18/6/en?format=image");
put("C18/Survivor", "https://api.scryfall.com/cards/tc18/12/en?format=image");
put("C18/Thopter/1", "https://api.scryfall.com/cards/tc18/25/en?format=image");
put("C18/Thopter/2", "https://api.scryfall.com/cards/tc18/26/en?format=image");
put("C18/Thopter/3", "https://api.scryfall.com/cards/tc18/8/en?format=image");
put("C18/Worm", "https://api.scryfall.com/cards/tc18/18/en?format=image");
put("C18/Zombie", "https://api.scryfall.com/cards/tc18/9/en?format=image");
```

**Comment by rikimaru1234321** [comment]
> 
> 
> File in root folder must be deleted

Can you tell me more specific?

**Comment by JayDi85** [comment]
shot_190823_151430

**Pull request merged**